content/departments/product-engineering/engineering/enablement/dev-experience/index.md

```diff
@@ -71,13 +71,60 @@ To learn more about contributing to `sg`, check out the [contribution guide](htt
 
 ### Planning
 
+This section describes how the DevX team currently conducts planning.
 Planning can include quarterly planning, project scoping, requirements gathering, RFC-writing, or any combination of the above.
 
-The DevX team uses GitHub issues with the [`team/devx` and `planning` labels](https://github.com/sourcegraph/sourcegraph/issues?q=sort%3Aupdated-desc+is%3Aissue+label%3Aplanning+label%3Ateam%2Fdevx) on our [GitHub board](https://github.com/orgs/sourcegraph/projects/212) to track any significant planning efforts.
-Loose documents, Slack discussions, [meetings](#meetings), etc. can be used, but they should all be captured within and/or linked from the relevant planning issue.
-This makes it easier to teammates and stakeholders to pick up context around the state of planning, and helps represent the often significant time investment involved with robust planning efforts.
+For any major body of work:
 
-Once a roadmap or scope has been finalized, the appropriate tasks should be created, consolidated under a [tracking issue](../../process/tracking_issues.md) where helpful.
+1. A [work lead](#work-leads) is assigned.
+2. A [planning issue](#planning-issues) is created, and relevant planning work is done.
+3. A [tracking issue](#tracking-issues) is created, tracking tasks required for completing the work.
+
+#### Work leads
+
+A **work lead** is the person assigned to lead a body of work, including the planning of said work.
+If there are questions about priority order, scope change, etc. the work lead helps answer questions/provide updates.
+They have special responsibilities regarding [planning issues](#planning-issues) and [tracking issues](#tracking-issues).
+
+#### Planning issues
+
+Planning starts with a **planning issue**. Planning issues are GitHub issues with the [`team/devx` and `planning` labels](https://github.com/sourcegraph/sourcegraph/issues?q=sort%3Aupdated-desc+is%3Aissue+label%3Aplanning+label%3Ateam%2Fdevx) on our [GitHub board](https://github.com/orgs/sourcegraph/projects/212) to track any significant planning efforts. A planning issue documents the planning process - links to RFC(s), discussions, artefacts like scratch documents, [meetings](#meetings), decisions, etc. as planning progresses.
+
+> NOTE: An updated and well-documented planning issue makes it easier to teammates and stakeholders to pick up context around the state of planning, and helps represent the often significant time investment involved with planning efforts.
```

Comment:
This is great. I wonder if it's worth saying something explicit like, "A stakeholder should be able to look at a planning issue and get a high-level understanding of the problem we're trying to solve and the steps we're taking to address it, without having to read through the linked resources." I want to make sure we avoid having the planning issue just be a list of aggregated links.